src/templates/kubernetes/kafka/kafka-deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kafka-broker
+  labels:
+    app: kafka
+    component: kafka-broker
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kafka
+  template:
+    metadata:
+      labels:
+        app: kafka
+        component: kafka-broker
+    spec:
+      containers:
+      - name: kafka
+        image: wurstmeister/kafka:2.12-2.3.0
```

Comment:
At some point of time, we might want to move all these explicit versioning into configuration parameters.

Reply:
Yeah true, I just touched those parts which always need a change during a build.

Reply:
ok, we can revisit in an iteration.